backend/routes/domains.js

```diff
@@ -31,6 +33,32 @@ module.exports = function (router) {
         domain: req.body.domain,
         status: ShopDomainStatuses.Pending
       })
+
+      // Re-configure CDN to pick up domain changes
+      const deployments = await ShopDeployment.findAll({
+        where: {
+          shopId: req.shop.id
+        },
+        limit: 1,
+        order: [['createdAt', 'DESC']]
```

Comment:
nit: order by id is more performant since there is no DB index on createdAt

Reply:
IDs technically aren't guaranteed to be sequential and is generally considered bad form.

Also, index performance boosts to sorting [are kind of minimal for large datasets and sometimes negative, relying more on disk IO instead of memory/CPU, causing disk thrashing on standard disks](https://www.postgresql.org/docs/9.3/indexes-ordering.html).  And considering the size of the dataset here, I think we're fine.

Reply:
Agreed data set is small so we don't need to worry.

Regarding "IDs technically aren't guaranteed to be sequential and is generally considered bad form", ids are generated by a sequence and therefore are guaranteed to be monotonically increasing, which is what we care about for ordering I think.

Reply:
Sequences can wrap.  INSERTs can be done with explicit IDs skipping sequences entirely.  While these conditions may be unlikely for our dataset, they're still considered bad form because of that.  ID ordering is implicit, while the createdAt date is explicit.